charts/feathr-online/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: feathr-online-config
+  namespace: default
+data:
+  lookup.json: {{ .Files.Get "conf/lookup.json" | toPrettyJson }}
+  pipeline.conf: {{ .Files.Get "conf/pipeline.conf" | indent 4 }}
```

Comment:
`"conf/lookup.json"` and `"conf/pipeline.conf"` should not be hard coded in template. They should be exposed for passing thru, lookup.json is optional while pipeline.conf is required